examples/llm-eval/lm-eval-harness/lm_eval_tensorrt_llm.py

```
@@ -103,10 +117,27 @@
                 pytorch_config_params["moe_backend"] = self.moe_backend
                 print(f"Info: moe_backend is set to {self.moe_backend}")
 
+            if self.backend == 'pytorch':
+                if hasattr(PyTorchConfig, "moe_backend"):
+                    pytorch_config_params["moe_backend"] = self.moe_backend
+                    print(f"Info: moe_backend is set to {self.moe_backend}")
+                pytorch_config_params["attn_backend"] = attn_backend.upper()
+                print(f"Info: attn_backend is set to {attn_backend}")
+                pytorch_config = PyTorchConfig(**pytorch_config_params)
+            elif self.backend == 'autodeploy':
+                assert self.max_context_length is not None, "max_context_length must be specified for autodeploy backend"
```

Ruff (0.12.2)

Line too long (121 > 120)

(E501)